tools/webpack-core/src/utils/resolve-recurse/resolve.js

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+//resolve.js
+
+var Promise = require('promise');
+var findup = Promise.denodeify(require('findup'));

```

Comment:
If we are making changes to this though, there's definitely no need to pull in `promise`.
You can use [util.promisify](https://nodejs.org/docs/latest-v12.x/api/util.html#util_util_promisify_original) instead of `Promise.denodeify`.

Reply:
On line 140 in the same file what would be the util equivalent of nodeify ?

Reply:
I'd just change the signature of `resolveRecurse` - remove the option to pass in a `done` callback and make it always an `async` function.

Reply:
Okay got it